src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static int MAX_SIZE = 100;
+    public static Task[] list = new Task[MAX_SIZE];
+    private static int LIST_COUNTER = 0;
+
+    /**
+     * Loop through all possible commands
+     * Possible Commands:
+     * Todo, event, Deadline, help, list, done, bye
+     */
+    public static void loop() {
+        Scanner line = new Scanner(System.in);
+        while (line.hasNextLine()) {
+            String input = line.nextLine();
+            if(LIST_COUNTER == 100){
+                error(6);
+                return;
+            }
+            if (input.equalsIgnoreCase("bye")) {
+                PrintOutput.printExit();
+                System.exit(1);
+            } else if (input.equalsIgnoreCase("help")) {
+                System.out.println(PrintOutput.HELP_MESSAGE);
+            } else if (input.toLowerCase().startsWith("done")) {
+                doneTask(input);
+            } else if (input.equalsIgnoreCase("list")) {
+                printList();
+            } else if (input.toLowerCase().startsWith("todo")){
+                addTodo(input);
+            } else if (input.toLowerCase().startsWith("deadline")){
+                addDeadline(input);
+            } else if (input.toLowerCase().startsWith("event")){
+                addEvent(input);
+            } else {
+                System.out.println("¯\\_(ツ)_/¯ That is an invalid command!");
+                System.out.println("Enter \"HELP\" for commands!");
+                PrintOutput.printBorder();
+            }
+        }
+    }
+
+    /**
+     * Main Function
+     * To Print Greeting as Default then loop function
+     * @param args - argument
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        PrintOutput.printGreeting();
+        loop();
+    }
+
+    /**
+     * Add new task to Timetable
+     * @param input - name of task
+     */
+    public static void addTodo(String input) {
+        if(input.equals("todo")){
+            error(5);
+            return;
+        }
+        String command = input.substring(5);
+        if(!command.isBlank()){
+            Todo t = new Todo(command);
+            list[LIST_COUNTER] = t;
+            LIST_COUNTER += 1;
+            System.out.println("I have added [" + t.getType() + "]["
+                + t.getStatusIcon() + "] \""
+                + t.getName() + "\" " + "to the List!");
```

Comment:
Indentation for wrapped lines should be 8 spaces. See standard's `Indentation for wrapped lines should be 8 spaces` section. (same applies to all wrapped lines below)